modules/calcite/src/test/java/org/apache/ignite/testsuites/IgniteCalciteTestSuite.java

```diff
@@ -38,7 +38,7 @@
     SqlDdlParserTest.class,
     IgniteSqlFunctionsTest.class,
 
-    ScriptTestSuite.class,
+    //ScriptTestSuite.class,
```

Comment:
Did you forgot to uncomment this before push?

Reply:
yes i did ) thanks